tests/components/nsw_fuel_station/test_sensor.py

```diff
@@ -104,7 +106,10 @@ def test_setup(self):
     )
     def test_sensor_values(self):
         """Test retrieval of sensor values."""
-        assert setup_component(self.hass, sensor.DOMAIN, {"sensor": VALID_CONFIG})
+        assert setup_component(self.hass, sensor.DOMAIN,
+                               {"sensor": VALID_CONFIG})
 
-        assert "140.0" == self.hass.states.get("sensor.my_fake_station_e10").state
-        assert "150.0" == self.hass.states.get("sensor.my_fake_station_p95").state
+        assert "140.0" == self.hass.states.get(
+            "sensor.my_fake_station_e10").state
+        assert "150.0" == self.hass.states.get(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4639533)